packages/carousel/src/Carousel.stories.js

```diff
@@ -179,10 +184,38 @@ AlternateButtonColors.args = {
   visibleSlides: 3,
   showDots: false,
   showForwardBackBtns: true,
-  buttonColorProps: {
+  buttonStyles: {
+    buttonBackground: 'primary.base',
+    buttonColor: 'text.lightest',
+    buttonHoverBackground: 'primary.dark',
+    buttonHoverColor: 'text.lightest',
+  },
+}
+
+const TwoColumLayoutTemplate = (args) => (
+  <Container maxWidth={1280}>
+    <Flex width={1} flexDirection={['column', null, null, 'row']}>
+      <Box width={[1, null, null, 2 / 3]}>
+        <Carousel {...args}>{renderCards()}</Carousel>
+      </Box>
+      <StyledMockBox width={[1, null, null, 1 / 3]} textAlign='center'>
+        MOCK SUMMARY OF CHARGES
+      </StyledMockBox>
+    </Flex>
+  </Container>
+)
+export const TwoColumnLayout = TwoColumLayoutTemplate.bind({})
+TwoColumnLayout.args = {
+  showDots: false,
+  showForwardBackBtns: true,
+  sideButtonMargin: '-23px',
+  buttonStyles: {
     buttonBackground: 'primary.base',
     buttonColor: 'text.lightest',
     buttonHoverBackground: 'primary.dark',
     buttonHoverColor: 'text.lightest',
+    buttonWidth: '48px',
+    buttonHeight: '48px',
   },
+  isTwoColumnLayout: true,
```

Comment:
I don't think that this prop is needed, you can override the `visibleSlides` prop, which should allow you to do this

Reply:
I removed `isTwoColumnLayout` prop. Your suggested change worked for my use case and I ended up using `visibleSlides={[2.1, 1, 1]}`